src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/net461/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -0,0 +1,47 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)..\common.targets" />
+  <Target
+      Name="MvcRazorPrecompile"
+      DependsOnTargets="_ResolveInputArguments"
+      Inputs="$(MSBuildThisFileFullPath);@(MvcRazorFilesToCompile);@(IntermediateAssembly);@(DocFileItem);@(_DebugSymbolsIntermediatePath);@(ReferencePath);$(MSBuildAllProjects);"
+      Outputs="$(_MvcRazorOutputFullPath)">
+
+    <PropertyGroup Condition="'$(MvcRazorRunCommand)'==''">
+      <MvcRazorRunCommand Condition="'$(PlatformTarget)'=='x86'">$(OutputPath)$(MSBuildThisFileName)-x86.exe</MvcRazorRunCommand>
+      <MvcRazorRunCommand Condition="'$(PlatformTarget)'!='x86'">$(OutputPath)$(MSBuildThisFileName).exe</MvcRazorRunCommand>
+    </PropertyGroup>
+
+    <CallTarget Targets="_CreateResponseFileForMvcRazorPrecompile" />
+
+    <ItemGroup Condition="'$(PlatformTarget)'=='x86'">
+      <FilesToCopy Include="$(OutputPath)$(AssemblyName).exe.config">
+        <Destination>$(OutputPath)$(MSBuildThisFileName)-x86.exe.config</Destination>
+      </FilesToCopy>
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(PlatformTarget)'!='x86'">
+      <FilesToCopy Include="$(OutputPath)$(AssemblyName).exe.config" Condition="">
+        <Destination>$(OutputPath)$(MSBuildThisFileName).exe.config</Destination>
+      </FilesToCopy>
+    </ItemGroup>
+
+    <Copy
+      SourceFiles="@(FilesToCopy)"
+      DestinationFiles="%(Destination)" />
+
+    <Message
```

Comment:
You can remove this log message as it doesn't really contain any useful information.